charts/catalog/Chart.yaml

```diff
@@ -1,3 +1,4 @@
 name: catalog
 description: service-catalog API server and controller-manager helm chart
-version: 0.1.24
+version: 0.1.25
```

Comment:
This is bumped manually during our releases. You don't need to touch it here.

See https://github.com/kubernetes-incubator/service-catalog/pull/2192/files for what a release looks like.

Reply:
OK, I think i'll open another issue to allow these to be changed independently. As is, the chart leaves svc cat in a broken state of someone turns on the feature gate, so we should probably do another release (which makes me a little sad)